doc/nrf/releases_and_maturity/releases/release-notes-2.5.0.rst

```diff
@@ -0,0 +1,1240 @@
+
+* Updated:
+
+  * Set the max compiled-in log level to ``warning`` for the USB HID class (:kconfig:option:`CONFIG_USB_HID_LOG_LEVEL_CHOICE`) and reduce the log message levels used in the :ref:`nrf_desktop_usb_state_pm` source code.
+    This is done to avoid flooding logs during USB state changes.
+  * If the USB state is set to :c:enum:`USB_STATE_POWERED`, the :ref:`nrf_desktop_usb_state_pm` restricts the power down level to the :c:enum:`POWER_MANAGER_LEVEL_SUSPENDED` instead of requiring :c:enum:`POWER_MANAGER_LEVEL_ALIVE`.
+    This is done to prevent the device from powering down and waking up multiple times when an USB cable is connected.
+  * Disabled ``CONFIG_BOOT_SERIAL_IMG_GRP_HASH`` in MCUboot bootloader release configurations of boards that use nRF52820 SoC.
+    This is done to reduce the memory consumption.
+  * To improve the accuracy, the generation of simulated movement data in the :ref:`nrf_desktop_motion` now uses a timestamp in microseconds based on the cycle count (either :c:func:`k_cycle_get_32` or :c:func:`k_cycle_get_64` function depending on the :kconfig:option:`CONFIG_TIMER_HAS_64BIT_CYCLE_COUNTER` Kconfig option).
+  * Aligned Kconfig option names in the :ref:`nrf_desktop_motion` implementation that generates motion from button presses.
+    The Kconfig options defining used key IDs are prefixed with ``CONFIG_MOTION_BUTTONS_`` instead of ``CONFIG_MOTION_`` to ensure consistency with configuration of other implementations of the motion module.
+  * The :ref:`nrf_desktop_ble_scan` no longer stops Bluetooth LE scanning when it receives :c:struct:`hid_report_event` related to a HID output report.
+    Sending HID output report is triggered by a HID host.
+    Scanning stop may lead to an edge case where the scanning is stopped, but there are no peripherals connected to the dongle.
+
+Thingy:53: Matter weather station
+---------------------------------
+
+* Added support for the nRF7002 Wi-Fi expansion board.
+
+Matter Bridge
+-------------
+
+* Added:
+
+  * The :ref:`Matter bridge <matter_bridge_app>` application.
+  * Support for the Bluetooth LE bridged devices.
+  * Support for bridging of the Bluetooth LE Environmental Sensor (ESP).
+  * Support for performing Device Firmware Upgrade (DFU) over Bluetooth LE using Simple Management Protocol (SMP).
+
+Samples
+=======
+
+Bluetooth samples
+-----------------
+
+* :ref:`direct_test_mode` sample:
+
+  * Added:
+
+    * Support for the nRF52840 DK.
+    * Experimental support for the HCI interface.
+
+  * Updated:
+
+    * Aligned timers' configurations to the new nrfx API.
+    * Extracted the DTM radio API from the transport layer.
+    * Added support for the radio fast ramp-up feature.
+      This feature is enabled by default.
+
+* :ref:`peripheral_hids_keyboard` sample:
+
+  * Fixed an interoperability issue with iOS devices by setting the report IDs of HID input and output reports to zero.
+
+* :ref:`fast_pair_input_device` sample:
```

Comment:
@alstrzebonski, please also take a look at the Fast Pair sample section in NCS 2.5.0 release notes